next-cmake/CMakeLists.txt

```
@@ -0,0 +1,464 @@
+# Copyright Advanced Micro Devices, Inc., or its affiliates.
+# SPDX-License-Identifier:  MIT
+
+cmake_minimum_required(VERSION 3.25.2)
+project(
+  rocblas
+  VERSION 5.1.0
+  LANGUAGES CXX)
+
+if(POLICY CMP0135)
+  cmake_policy(SET CMP0135 NEW)
+endif()
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+# Set cpack license file here to avoid rocm-cmake warning
+set(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/../LICENSE.md")
+
+include(rocblas_target_configure_sanitizers)
+include(query_git_hash)
+include(FetchROCmCMake)
+include(CMakeDependentOption)
+include(CMakePackageConfigHelpers)
+include(GenerateExportHeader)
+include(GNUInstallDirs)
+
+rocm_setup_version(VERSION ${PROJECT_VERSION})
+set(rocblas_SOVERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}")
+
+set(HIP_RUNTIME_MINIMUM
+    "4.5.0"
+    CACHE STRING "Minimum HIP runtime version required")
+
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  if(WIN32)
+    set(CMAKE_INSTALL_PREFIX
+        "C:/hipSDK"
+        CACHE PATH "Install path prefix." FORCE)
+  else()
+    set(CMAKE_INSTALL_PREFIX
+        "/opt/rocm"
+        CACHE PATH "Install path prefix." FORCE)
```

```suggestion
    set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix." FORCE)
```
Would match hipBLASLt's next-cmake.

---

I've applied cmake-format to the entire next-cmake directory. This formatting difference comes from that. If we would like, I can look into establishing a proper cmake-format configuration file so it applies a format that we like.

---

I would prefer fewer line breaks but it isn't too urgent.